tests/Tests/400__ods_dashboard/400__ods_dashboard.robot

```diff
@@ -729,7 +738,37 @@ Check And Uninstall Operator In Openshift
     END
     Close All Browsers
 
+Check Application Switcher Links To Openshift Cluster Manager
+    [Documentation]    Checks for HTTP status of OCM link in application switcher
+    ${cluster_id}=    Get Cluster ID
+    ${cluster_id}=    Remove String    ${cluster_id}    "
```

Comment:
Can " be removed inside `Get Cluster ID`?

Reply:
checking this one